macros/AddonSidebar.ejs

```diff
@@ -58,6 +58,7 @@ var text = mdn.localStringMap({
     'WebExtensions/Embedded_WebExtensions': 'Embedded WebExtensions',
     'WebExtensions/Comparison_with_the_Add-on_SDK': 'Comparison with the Add-on SDK',
     'WebExtensions/Comparison_with_XUL_XPCOM_extensions': 'Comparison with XUL/XPCOM extensions',
+    'WebExtensions/differences_between_extensions_API_implementations': 'Extension API differences',
```

Comment:
This should be <q>Differences between API implementations</q> located at [WebExtensions/differences_between_API_implementations](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/differences_between_API_implementations).

Reply:
Updated

Reply:
The URL still points to the old location.

Reply:
Sorry, URL fixed